frame/nomination-pools/src/migration.rs

```diff
@@ -384,3 +384,73 @@ pub mod v2 {
 		}
 	}
 }
+
+pub mod v3 {
```

Comment:
would be nice to have a comment what this migration about

Reply:
done